packages/block-editor-tools/src/hooks/use-debounce/README.md

```diff
@@ -17,7 +19,7 @@ const MyComponent = () => {
   return (
     <>
       <TextControl
-        label={__('Set Value', 'alley-scripts')}
```

Comment:
> Users should be using their textdomain rather than relying on the WP global textdomain

- I believe users copying this snippet can still do that;
- Using `your-textdomain-here` or nothing, which defaults to `default`, communicates the same thing: you are copying this and you should update with your projects text domain;
- This value is not an official value, saved by us. If someone copies and forgets to update it, the parser will create a translation which is not ours.

My main rationale was: it is the responsibility of the snippet copier to update this to the right value. The Board Connect loc is an example of how easily it is to copy the snippet and forget to change the text domain.

Some [projects](https://developer.buddypress.org/bp-rest-api/extending-the-bp-rest-api/) don't even add it in their documentation to avoid this very issue.

cc: @kevinfodness

Reply:
I feel strongly that we should include the textdomain with a placeholder to indicate to implementers that there is a second parameter to that function and that it needs to be filled out. I think it's more obvious to implementers that they need to update the value if there is a default one set than if the second parameter is omitted entirely. To that end, I'd recommend using `your-textdomain-here` as it's clear that it needs to be updated by the implementer.

Reply:
Will add this change. Tks! :)